tests/plan/terraform/versions.tf

```diff
@@ -12,7 +12,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 5.0"
+      version = "<5.67.0"
```

Comment:
why is this pinned?

Reply:
There was a bug with the latest release that was breaking the build. Will find it and link